libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/signboard/api/MapboxSignboardApi.kt

```diff
@@ -17,7 +17,7 @@ import com.mapbox.navigation.utils.internal.ThreadController
 import kotlinx.coroutines.launch
 
 /**
- * Implementation of [SignboardApi] allowing you to generate signboard for select maneuvers.
+ * Mapbox Signboard Api allows you to generate signboard for select maneuvers.
  * @property accessToken String
  */
 class MapboxSignboardApi(
```

Comment:
Same as above:

> Should we add a note here remarking that a special `accessToken` is needed for this feature to work?

Reply:
This will be handled in the docs as we did in v1.x